### ISSUE TITLE
mismatched types when attempting to import helios and ethers into the same project

### ISSUE THREAD

**Issue by naps62** [a16z/helios#127]
Trying the sample code in the `README` leads me to a few compilation errors such as this:

```
error[E0308]: mismatched types
   --> /home/naps62/.cargo/git/checkouts/helios-b3bc464b79507b80/e4071fe/execution/src/evm.rs:256:13
    |
255 |         Ok(Some(AccountInfo::new(
    |                 ---------------- arguments to this function are incorrect
256 |             account.balance,
    |             ^^^^^^^^^^^^^^^ expected struct `primitive_types::U256`, found struct `ethers::types::U256`
    |
    = note: struct `ethers::types::U256` and struct `primitive_types::U256` have similar names, but are actually distinct types
```

I tried overriding `ethers` to use the same fork specified in this repo, but that seems to have changed nothing

**Comment by ncitron** [comment]
Hmmm I've noticed that sometimes when people build Helios, especially when importing it from elsewhere, it sometimes struggles to match the `primitive_types` crates used by `ethers-rs` and `revm`. Will dig into why this is happening.

**Comment by refcell** [comment]
Fixed in #129
